nmostesting/suites/IS1101Test.py

```diff
@@ -1769,3 +1750,298 @@ def test_04_02_02(self, test):
+                if not input["edid_support"]:
+                    continue
+
+                valid, response = self.do_request("GET", self.compat_url + "inputs/" + inputId + "/edid/effective")
+                if not valid or response.status_code != 200:
+                    return test.FAIL("Unexpected response from "
+                                     "the Stream Compatibility Management API: {}".format(response))
+
+                effective_edid_before = response.content
+
+                valid, response = self.do_request("DELETE", self.compat_url + "inputs/" + inputId + "/edid/base")
+                if not valid or response.status_code != 204:
+                    return test.FAIL("Unexpected response from "
+                                     "the Stream Compatibility Management API: {}".format(response))
+
+                valid, response = self.do_request("GET", self.compat_url + "inputs/" + inputId + "/edid/effective")
+                if not valid or response.status_code != 200:
+                    return test.FAIL("Unexpected response from "
+                                     "the Stream Compatibility Management API: {}".format(response))
+
+                if response.content == effective_edid_before:
+                    return test.FAIL("Effective EDID doesn't change when Base EDID changes")
+
+                inputs_tested.append(inputId)
+
+            except json.JSONDecodeError:
+                return test.FAIL("Non-JSON response returned from Node API")
+            except KeyError as e:
+                return test.FAIL("Unable to find expected key: {}".format(e))
+
+        if len(inputs_tested) > 0:
+            return test.PASS()
+        else:
+            return test.UNCLEAR("Not tested. No inputs with EDID support found.")
+
+    def test_06_06(self, test):
+        """
+        Immediate activation of a receiver without a transport file switches the state of the receiver to 'unknown'
+        """
+
+        if len(self.receivers) == 0:
+            return test.UNCLEAR("Not tested. No receivers found.")
+
+        receivers_tested = []
+        warn = ""
+        for receiverId in self.receivers:
+            valid, response = self.do_request("GET", self.node_url + "receivers/" + receiverId)
+            if not valid or response.status_code != 200:
+                return test.FAIL("Unexpected response from the Node API: {}".format(response))
+
+            try:
+                receiver = response.json()
+                if not receiver["transport"].startswith("urn:x-nmos:transport:rtp"):
+                    continue
+
+                url = "single/receivers/{}/staged".format(receiverId)
+                data = {"sender_id": None, "transport_file": {"data": None, "type": None}}
+                valid, response = self.is05_utils.checkCleanRequestJSON("PATCH", url, data)
+                if not valid:
+                    return test.FAIL("Receiver {} rejected staging of absent SDP file: "
```

Comment:
I don't think there is any requirement in MOST IS-05 for a Receiver to accept a PATCH without an SDP transport file for RTP transport. I would return a Could Not Test instead of a Fail as it is a legitimate restriction.

Reply:
OK, removed test 6.